base/number.jl

```diff
@@ -287,7 +287,12 @@ map(f, x::Number, ys::Number...) = f(x, ys...)
     zero(x)
     zero(::Type)
 
-Get the additive identity element for the type of `x` (`x` can also specify the type itself).
+Get the additive identity element for `x`. If the additive element can be known based
+on the type alone, then a type may be given as an argument to `zero`.
```

Comment:
```suggestion
Get the additive identity element for `x`. If the additive identity can be deduced
from the type alone, then a type may be given as an argument to `zero`.
```

Reply:
BTW I find the difference in phrasing for the docstring of `one` and `zero` that existed before this PR odd. Some minor things ("Get" vs. "Return") are fine, but why is it "*the* additive identity *element*" but "a multiplicative identity" ?

Ironically, for `Int` the multiplicative identity is unique, so "the" would be warranted, while for e.g. `Float64` the additive identity is a bit ambiguous... both `0.0` or `-0.0` satisfy the identity `zero(x)+x == x*zero(x) == x` (but only because `0.0 == -0.0` holds).

BTW the fact that the axiom `one(x)*x == x*one(x) == x` is there, but not the additive counterpart is another odd difference...

(All that of course is not meant to hold back this PR, just me rambling)